Dockerfile

```diff
@@ -36,7 +36,8 @@ WORKDIR /code
 # NOTE: requirements.txt not likely to change between dev builds
 COPY requirements.txt .
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends netcat vim-tiny jq python3-dev xmlsec1 cron && \
+    apt-get install -y --no-install-recommends \
+    build-essential default-libmysqlclient-dev  libpq-dev netcat vim-tiny jq python3-dev xmlsec1 cron && \
```

Comment:
This reminds me, I'd like to create an issue where we remove the cron. I wonder how Unizin is running this though.